spec/requests/users_spec.rb

```diff
@@ -22,7 +22,8 @@
     end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/205986537-db6a53ce-4d04-4d4d-b372-ba3185297da3.png)

- [ ] One of your test is failing kindly fix that by dropping your database and by creating in again :blush:

Reply:
Hi @HAMMAS-SALEEM , thank you so much for your feedback. I already solved it 👍🏼